shared/util/file.native.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+// TODO
+function tmpFile (suffix: string): string {
+  return ''
```

Comment:
I'd `throw new Error('Unimplemented`) or something so it breaks if someone accidentally uses, ditto below